pages/wi/voting.js

```diff
@@ -198,6 +198,10 @@ const VotingStart = ({ user, loading }) => {
                       </strong>{' '}
                       for more info on sharing your awesome.
                     </li>
+                    <li>
+                      {/* This should not be hard coded */}
```

Comment:
Seriously

Reply:
What about using the milestone?

Reply:
There is no key/id on a milestone so I would have to magic-string the text which is pretty much same as hard-coding the date.

Reply:
Perhaps, but it allows the date to be changed without redeploying, etc.

Reply:
The ability to query for a specific milestone to get its date is something we've all talked about several times, but has yet to rise to the top of the heap in priorities. For the time being, if we want this (or any other specific milestone) date on a page it has to be hard coded.

Reply:
I'd suggest we just pull that out to a "constants" like file now and reference that. Then when the right time comes all we need to do is update one space regardless.